veashi-scanner/globals.css

```
@@ -0,0 +1,130 @@
+@import url('https://fonts.googleapis.com/css2?family=DM+Sans:opsz,wght@9..40,400;9..40,500;9..40,600;9..40,700&family=JetBrains+Mono:wght@400;500;600&display=swap');
+@import "./node_modules/@kleros/ui-components-library/dist/assets/theme.css";
+@source "./node_modules/@kleros/ui-components-library";
```

Stylelint (17.12.0)

[error] Unexpected unknown at-rule "`@source`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)